src/app/src/test/kotlin/org/imperial/mrc/hint/integration/BaselineTests.kt

```diff
@@ -1,41 +1,32 @@
 package org.imperial.mrc.hint.integration
 
 import org.assertj.core.api.Assertions.assertThat
+import org.imperial.mrc.hint.helpers.JSONValidator
 import org.imperial.mrc.hint.helpers.createTestHttpEntity
-import org.imperial.mrc.hint.helpers.tmpUploadDirectory
-import org.junit.jupiter.api.AfterEach
 import org.junit.jupiter.api.Test
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.boot.test.web.client.TestRestTemplate
 import org.springframework.boot.test.web.client.getForEntity
 import org.springframework.boot.test.web.client.postForEntity
 import org.springframework.http.HttpStatus
-import java.io.File
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-class BaselineTests(@Autowired val testRestTemplate: TestRestTemplate) {
-
-    @AfterEach
-    fun tearDown() {
-        File(tmpUploadDirectory).deleteRecursively()
-    }
+class BaselineTests(@Autowired val testRestTemplate: TestRestTemplate): IntegrationTests() {
 
     @Test
     fun `can get baseline data`() {
         val entity = testRestTemplate.getForEntity<String>("/baseline/")
         assertThat(entity.statusCode).isEqualTo(HttpStatus.OK)
-        assertThat(entity.body!!).isEqualTo("{\"pjnz\": null}")
```

Comment:
We can't yet validate this endpoint against the schema because the schema is too prescriptive, see https://dideimperial.slack.com/archives/GL5N84ZFU/p1566920573096200